### ISSUE TITLE
SerialLib.jl was deleted

### ISSUE THREAD

**Issue by tkelman** [JuliaLang/METADATA.jl#3592]
http://github.com/windcoon/SerialLib.jl no longer exists. What's the right thing to do about that, delete all mention of it from metadata? Or add infeasible bounds on the one tag? Ref https://github.com/JuliaLang/METADATA.jl/pull/3195, looks like it was mostly just a copy of Arduino.jl that maybe should have been looked at more closely before merging in the first place.

**Comment by jiahao** [comment]
Purge. It's completely impossible to reason about now that the entire repo is gone.

**Comment by sjkelly** [comment]
I've been meaning to consolidate the Serial port access packages (Arduino
and SerialPorts AFAIK) and have better examples. I have gotten iJulia and
Gadfly running on the RasPi2 and Intel Edison so there are some ripe
opportunities for data logging, control, and processing for the maker and
research community.

**Comment by johnmyleswhite** [comment]
That sounds awesome.

**Comment by tkelman** [comment]
Worth creating a JuliaEmbedded org at some point? We've got a bunch of Odroids in my lab that are using inferior Python tools just to avoid compiling all of julia, and some colleagues have been asking about RPi2 as well.

**Comment by sjkelly** [comment]
The Edison can run 32 bit Julia binaries fine on wheezy. Some instructions are here: https://github.com/sjkelly/mraa.jl

The RasPi2 needs Jessie, which I got through someone at Collabora (it was the first Google result at the time). I have yet to try the new Raspian Jessie distribution. 0.4 compiles flawlessly on it, and I was able to run the binary on other chips as well. I'll look into it more this weekend and see what could be done to get more frequent builds going and hosted somewhere.
